go-client/admin/client.go

```diff
@@ -33,51 +34,101 @@ import (
 // Client provides the administration API to a specific cluster.
 // Remember only the superusers configured to the cluster have the admin priviledges.
 type Client interface {
-	CreateTable(ctx context.Context, tableName string, partitionCount int, successIfExist_optional ...bool) error
+	Close() error
 
-	DropTable(ctx context.Context, tableName string) error
+	GetTimeout() time.Duration
+	SetTimeout(timeout time.Duration)
 
-	ListTables(ctx context.Context) ([]*TableInfo, error)
-}
+	CreateTable(tableName string, partitionCount int32, replicaCount int32, envs map[string]string, maxWaitSeconds int32, successIfExistOptional ...bool) (int32, error)
```

Comment:
It would be better to describe what does `maxWaitSeconds`, I doubt if there is any difference with `timeout`.

Reply:
It's necessary to add comments to explain this parameter. I'll add some comments.